content/articles/securing-mongodb-in-linux/index.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /securing-mongodb-in-linux/
+title: How to secure MongoDB in Linux
+description: This tutorial aims to show our readers how to protect their MongoDB setup by using some activation command lines in Linux.
+author: ali-benard
+date: 2022-02-03T00:00:00-17:00
+topics: [Security]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/securing-mongodb-in-linux/hero.jpg 
+    alt: Securing MongoDB in Linux Hero Image
+---
+### Content
+- [Prerequisites](#prerequisites)
+- [Introduction](#introduction)
+- [Check the environment](#check-the-environment)
+- [Securing MongoDB](#securing-mongodb)
+- [Create a user with system access](#create-a-user-with-system-access)
+- [Turn on authentication](#turn-on-authentication)
+- [Authentication Testing](#authentication-testing)
+- [Wrapping it up](#wrapping-it-up)
+
+### Prerequisites
+Before you start reading this tutorial, you need to have the following in your machine:
+- [Ubuntu](https://ubuntu.com/download/server) 20.05 server installed.
+- UFW-configured proxy server.
+- [MongoDB](https://www.mongodb.com/try/download/community) installed.
+
+### Introduction
+MongoDB is one of the most used modern web applications for a database. MongoDB is not secure by default. Once installed, one will have a difficult time before configuring it for authentication. Because people can browse, write, delete, or modify data on your server without ever needing to log in or authenticate. Securing the database is simple and may be completed in a few steps. In this tutorial, you will learn how to secure and activate authentication on MongoDB.
+
+### Check the environment
+To check if MongoDB is running on Linux kernel, we use the command below:
+```bash
+#call mongod and specify the path.
+mongod --dbpath = "C:\data\mongo"
+#or use this below to check.
+ps -e|grep 'Mongod'
+```
+If there comes MongoDB is already installed in your machine, the output will be similar to this:

```

Comment:
"If there comes MongoDB is already installed in your machine..."

This sentence is not clear.